pubsub/google/cloud/pubsub_v1/subscriber/policy/thread.py

```diff
@@ -85,9 +82,8 @@ def __init__(self, client, subscription, flow_control=types.FlowControl(),
             flow_control=flow_control,
             subscription=subscription,
         )
-        # Default the callback to a no-op; the **actual** callback is
-        # provided by ``.open()``.
-        self._callback = _do_nothing_callback
+        # The **actual** callback is provided by ``.open()``.
+        self._callback = None
```

Comment:
Are you testing `callable` before the callback is called?

Reply:
Not sure what you mean?